assets/src/js/admin/onboarding-wizard/utils/index.js

```diff
@@ -29,84 +38,62 @@ export const redirectToSetupPage = () => {
  * @since 2.8.0
  */
 export const saveSettingWithOnboardingAPI = ( setting, value ) => {
-	// Example shape of returned data
-	return {
-		setting,
-		value,
-	};
-
 	// Logic for connecting to the Onboarding API
-	// An object with action: 'save' and satting: ${value} is passed to the API
+	// An object with action: 'save' and setting: ${value} is passed to the API
 	// An object of the same shape should be returned to confirm the value was stored as expected
 	// Note: When the below code is actually implemented, the ${value} should be
 	// stringified (using qs library or JSON stringify).
 
 	// // Setup cancel token for request
-	// const CancelToken = axios.CancelToken;
-	// const source = CancelToken.source();
+	const CancelToken = axios.CancelToken;
+	const source = CancelToken.source();
+
+	axios.post( getAPIRoot() + 'give-api/v2/onboarding/settings', {
+		cancelToken: source.token,
```

Comment:
I think the `cancelToken` was in the boilerplate when I implemented the functionality, but I don't think it is necessary. I don't see a case where we will need to cancel the promise. Should we just remove it?

Reply:
I agree with you on this, I think we're fine to remove the cancel token logic. If we find a need to implement it, we know how to make that happen.